cloudflare.py

```diff
@@ -286,11 +288,13 @@ def desired(self):
 
     def diff(self):
         existing_tuples = {(record.type, record.name, record.content): record for record in self.existing()}
-        desired_tuples = {(record.type, record.name, record.content): record for record in self.desired()}
+        desired_tuples = {(record.type, record.name, record.content, record.managed): record for record in self.desired()}
 
```

Comment:
```suggestion
        desired_managed = {record.name: record.managed for record in self.desired()}
        
```

Reply:
and then we check for it before appending the ACTION_REMOVE 
```
            if key[1] in desired_managed and desired_managed[key[1]] == False:
                continue
```
(won't let me submit as a suggestion on that line)

Reply:
Now that I've committed those suggestions, maybe it will let you suggest it now?